j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/JsZipBundleTask.java

```diff
@@ -54,6 +54,7 @@ public Task resolve(Project project, Config config) {
                     Collections.emptyList(),
                     Collections.emptyMap(),
                     Collections.emptyList(),
+                    config.getTranslationsFile(),
```

Comment:
This won't work I think, it should error out actually - did you confirm that BUNDLE_JAR behaves when there is a translations file?

Reply:
So two notes, first of all you can never make calls on `config` inside the lambda - it should error out (and i'm surprised it doesn't even with a non-empty translations file... worth looking into, that means nulls don't affect the cache), and second this is a BUNDLE so you should just give it Optional.empty()